lib/task/export/csvExportTermUsageTask.class.php

```diff
@@ -102,6 +102,17 @@ private function determineTaxonomyId($options)
 
   private function exportToCsv($taxonomyId, $exportPath, $rowsUntilUpdate)
   {
+    // Offer to delete export file, if it already exists
```

Comment:
To avoid exportToCsv doing too many things, I'd consider moving this into its own function (private function checkReplaceFile()?), and maybe moving the check up into execute() before exportToCsv() is called. What do you think?